constructor/nsis/OptionsDialog.nsh

```diff
@@ -167,25 +213,36 @@ FunctionEnd
 Function AddToPath_OnClick
     Pop $0
 
+    # Sync UI with variable
+    ${NSD_GetState} $0 $1
+    ${If} $1 == ${BST_CHECKED}
+        StrCpy $INIT_CONDA 1
+    ${Else}
+        StrCpy $INIT_CONDA 0
+    ${EndIf}
+
     ShowWindow $Ana_AddToPath_Label ${SW_HIDE}
-    ${NSD_GetState} $0 $Ana_AddToPath_State
-    ${If} $Ana_AddToPath_State == ${BST_UNCHECKED}
+    ${If} $INIT_CONDA == 1
```

Comment:
The original coloring was actually correct. Adding `condabin` to `PATH` is pretty harmless. It should only be red when the mode is `classic`.